src/Microsoft.AspNetCore.Mvc.Core/breakingchanges.netcore.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "TypeId": "public class Microsoft.AspNetCore.Mvc.ApiControllerAttribute : Microsoft.AspNetCore.Mvc.ControllerAttribute, Microsoft.AspNetCore.Mvc.Internal.IApiBehaviorMetadata",
+    "Kind": "Removal"
```

Comment:
Is this a bug in API check?

Reply:
https://github.com/aspnet/BuildTools/issues/751